src/routes/userdata.py

```diff
@@ -586,7 +706,7 @@ def validate_signup():
 
     user = None
     try:
-        user = int__db.add_user(
+        user = int__db.add_google_user(
```

Comment:
Make sure to consolidate the user table from models.py with the user table within routes.py file